examples/learn-tensors-with-mcov/learn-tensors-with-mcov.py

```diff
@@ -0,0 +1,445 @@
+#
+#   \mathbf{T}_{1}(\{\mathbf{r}_i\}) = \sum_{\alpha = 1}^3 f_{\alpha}
+#   (\{\mathbf{r}_i\})\,\mathbf{q}_{\alpha},
+#
+# and :math:`\lambda=2` components as
+#
+# .. math::
+#
+#   \mathbf{T}_{2}(\{\mathbf{r}_i\}) = \sum_{\alpha = 1}^2 f_\alpha(\{\mathbf{r}_i\})
+#   \mathbf{q}_\alpha^{\widetilde{\otimes} 2}\,\,+ \sum_{\substack{\alpha_1,\alpha_2=
+#   1\\\alpha_1<\alpha_2}}^{3} g_{\alpha_1\alpha_2}(\{\mathbf{r}_i\})
+#   \big(\mathbf{q}_{\alpha_1}\widetilde{\otimes} \mathbf{q}_{\alpha_2}\big)_{2}.
+#
+# More generally, for any :math:`\lambda>2` we have:
+#
+# .. math::
+#
+#   \mathbf{T}_{\lambda} (\{\mathbf{r}_i\}) = \sum_{l= 0}^\lambda f_{l}
+#   (\{\mathbf{r}_i\}) \Big(\mathbf{q}_1^{\widetilde{\otimes} l}\widetilde{\otimes}
+#   {\mathbf{q}}_2^{\widetilde{\otimes}(\lambda-l)}\Big)_{\lambda}+
+#   \sum_{l=0}^{\lambda-1} g_{l} (\{\mathbf{r}_i\})
+#   \Big(\mathbf{q}^{\widetilde{\otimes} l}_1\widetilde{\otimes}
+#   \mathbf{q}_2^{\widetilde{\otimes} (\lambda-l-1)}\widetilde{\otimes}
+#   \mathbf{q}_3\Big)_{\lambda\mu}.
+#
+# 4. :math:`\lambda`-Correction Term
+# """"""""""""""""""""""""""""""""""
+#
+# Highly symmetric environments can lead to all-zero vector spherical expansion
+# components, which in turn would yield all-zero tensor features.
+# To correct this, we add a term based on the order :math:`\lambda` spherical expansion:
+#
+# .. math::
+#   \mathbf{T}_\lambda^{\rm corr}(\{\mathbf{r}_i\}) \;=\;
+#   \sum_{\beta=1}^{2\lambda+1}
+#   h_\beta(\{\mathbf{r}_i\})\,
+#   \sum_{z,n}W^{\rm corr}_{\beta,zn}\,\boldsymbol{\rho}_{z,n}^{l=\lambda}
+#
+# with learnable scalar functions :math:`h_\beta(\{\mathbf{r}_i\})`.
+#
+# 5. Scalar Network (SOAP-BPNN)
+# """""""""""""""""""""""""""""
+#
+# We first compute **SOAP powerspectrum** features:
+#
+# .. math::
+#
+#   p_{z_1z_2,n_1 n_2,l} = \bigl(\boldsymbol{\rho}_{z_1,n_1 l} \widetilde{\otimes}
+#   \boldsymbol{\rho}_{z_2,n_2 l}\bigr)_0
+#
+# and then apply a small, per-species multi-layer perceptron to map these features to
+# scalar coefficients :math:`f,g,h`.
+#
+# 6. Assembly and Global Output
+# """""""""""""""""""""""""""""
+#
+# Finally, we assemble the tensor:
+#
+# .. math::
+#
+#   \mathbf{T}_\lambda(\{\mathbf{r}_i\}) = \sum_{\beta=1}^{2\lambda+1}
+#   s_\beta(\{\mathbf{r}_i\}) \, \mathbf{B}^\lambda_\beta
+#   (\mathbf{q}_1,\mathbf{q}_2,\mathbf{q}_3) + \mathbf{T}_\lambda^{\mathrm{corr}}
+#   (\{\mathbf{r}_i\}),
+#
+# where :math:`\mathbf{B}^\lambda_\beta` is a shorthand for the basis tensors and
+# :math:`s_\beta` for the scalar coefficients. For global properties we sum over all
+# atoms.
+
+# %%
+# Training and evaluation of the model
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# We train the model with the ``metatrain`` package via its command-line interface.
+# To start training, we run
+#
+# .. code-block:: shell
+#    mtt train data/options.yaml
+#
+# The options file specifies the model architecture and the training parameters:
+#
+# .. literalinclude:: data/options.yaml
```

Comment:
This is not working, you should check why. I suspect that stuff in data/ is not copied over when the sphinx-gallery files are moved to the docs/examples folder.

Reply:
metatrain expects .yaml though, no? you can also (and maybe it's better) modify 
`src/generate-gallery.py:            "copyfile_regex": r".*\.(sh|xyz|cp2k|yml|png|zip)"`
to add yaml as one of the files that get copied over